### PR TITLE
Show a button to select address + frontend validation if default address is missing

### PR DESCRIPTION
**Pull request by indykoning**
See: https://github.com/rapidez/checkout-theme/pull/64
Dependent on https://github.com/rapidez/checkout-theme/pull/64/files#diff-54fd1a553d843584fafbe5181bad8ff684f46c5d52740602d5a50f0054ea6716

### CODE REVIEW

File: resources/views/components/address.blade.php [hunk 1]
```diff
@@ -6,37 +6,42 @@
...
+                    <template v-else>@lang('Address')</template>
+                @endif
+            </p>
+            <div class="flex flex-1 flex-wrap justify-between">
+                <ul class="flex flex-col gap-1">
+                    <li v-if="company">@{{ company }}</li>
+                    <li v-if="name">@{{ name }}</li>
+                    <li v-if="street">@{{ street }}</li>
+                    <li v-if="city">@{{ city }}</li>
+                    <li v-if="country === 'NL'">@lang('The Netherlands')</li>
```
Comment (JimmyHoenderdaal) [inline resources/views/components/address.blade.php:33]: Correct me if I'm wrong but this seems to be a bit weird to put into a package like this, why is this required?

Comment (royduin) [reply resources/views/components/address.blade.php:33]: Correct but it was already there, if it needs to become something else please create a separated PR 😇

Comment (JimmyHoenderdaal) [reply resources/views/components/address.blade.php:33]: I see, didn't know that it was already there 👍
